packages/node-bridge/launcher.js

```diff
@@ -87,19 +87,29 @@ function getVercelLauncher({
           bridge.listen();
         } else if (typeof listener === 'function') {
           Server.prototype.listen = originalListen;
+
           if (shouldAddHelpers) {
             bridge.setStoreEvents(true);
-            import(helpersPath).then(helper => {
-              const h = helper.default || helper;
-              const server = h.createServerWithHelpers(listener, bridge);
+          }
+
+          import(helpersPath).then(helper => {
```

Comment:
I added the new wrapper in `helpers.ts` mostly to add type-safety, but this means we have to import this in both cases of `shouldAddHelpers`. I could move that function in this file to avoid having to do that